From fc50476fc1b22299b90f1c0d6ce6a023743df3c1 Mon Sep 17 00:00:00 2001
From: Dmitry Smirnov <divis1969@gmail.com>
Date: Fri, 15 Apr 2016 22:51:05 +0300
Subject: [PATCH] Add MTK-specific APIs of the audio HAL

These APIs allow Engineering app to read/write audio data from/to NV

Change-Id: Ic1b59119081b41b8d1f59abfbf24a33b3d97a7a6
---
 include/hardware/audio.h | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/include/hardware/audio.h b/include/hardware/audio.h
index 22e7419..2170bfc 100644
--- a/include/hardware/audio.h
+++ b/include/hardware/audio.h
@@ -677,6 +677,12 @@ struct audio_hw_device {
     int (*listen_set_parameters)(struct audio_hw_device *dev,
                                  const char *kv_pairs);
 #endif
+    int (*dummy1)(void);
+    int (*dummy2)(void);
+    int (*dummy3)(void);
+    int (*dummy4)(void);
+    int (*set_audio_data)(struct audio_hw_device *dev, int par1, size_t len, void *ptr);
+    int (*get_audio_data)(struct audio_hw_device *dev, int par1, size_t len, void *ptr);
 };
 typedef struct audio_hw_device audio_hw_device_t;
 
